week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,13 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  return price*1.4;
+}
+function convertToBRL(price) {
+  return price*4.8 +price*4.8/100;
```

Comment:
We want 1% fee to be added in both conversions. How can we share the code so both of these functions end up adding 1% fee?